pkg/apis/servicecatalog/types.go

```diff
@@ -60,8 +60,31 @@ type ServiceBrokerSpec struct {
 	// CABundle is a PEM encoded CA bundle which will be used to validate a Broker's serving certificate.
 	// +optional
 	CABundle []byte
+
+	// RelistBehavior specifies the type of relist behavior the catalog should exhibit
+	// when relisting ServiceClasses available from a broker
+	RelistBehavior ServiceBrokerRelistBehavior
+
+	//RelistDuration is the frequency by which a controller will relist the broker
```

Comment:
There should be spaces after the `//` and all godoc comments for fields should end in periods.